pkg/customercard/request.go

```diff
@@ -1,5 +1,13 @@
 package customercard
 
 type Request struct {
-	Token string `json:"token"`
+	Issuer IssuerRequest `json:"issuer,omitempty"`
```

Comment:
não estamos usando ponteiros nas requests?
```suggestion
	Issuer *IssuerRequest `json:"issuer,omitempty"`
```

Reply:
Sim, corrigido.